readthedocs/doc_builder/config.py

```diff
@@ -2,19 +2,22 @@
 
 from os import path
 
-from readthedocs.config import BuildConfigV1, ConfigFileNotFound
+from readthedocs.config import BuildConfigV1
 from readthedocs.config import load as load_config
 from readthedocs.projects.models import ProjectConfigurationError
 
+from ..config.config import DefaultConfigFileNotFound
 from .constants import DOCKER_IMAGE, DOCKER_IMAGE_SETTINGS
 
 
-def load_yaml_config(version):
+def load_yaml_config(version, config_file=None):
```

Comment:
To keep consistency through the whole code, I'd use the same name here `readthedocs_yaml_path`, instead of just `config_file`.